src/rgw/driver/rados/rgw_bucket.cc

```diff
@@ -862,11 +861,9 @@ int RGWBucketAdminOp::sync_bucket(rgw::sal::Driver* driver, RGWBucketAdminOpStat
 }
 
 static int bucket_stats(rgw::sal::Driver* driver,
-			const std::string& tenant_name,
-			const std::string& bucket_name,
-			Formatter *formatter,
-                        const DoutPrefixProvider *dpp, optional_yield y)
-{
+                        const std::string& tenant_name,
```

Comment:
please avoid reformatting unchanged code - this is creating merge conflicts

Reply:
> please avoid reformatting unchanged code - this is creating merge conflicts

Sorry for the inconvenience.